patricia/novo1.md

```diff
@@ -0,0 +1,3 @@
+eae
+isso eh mto legaaaalll
+#EIJI_peido+
```

Comment:
What do you think about adding a blank space between # and the text?

Reply:
blank space added